pkg/database/models/process.go

```diff
@@ -103,8 +103,8 @@ const (
 type SearchKey string
 
 const (
-	KevinKey SearchKey = "AO6VqIdnT4z+h5mO9Ig3QmvGMwTgsJeZ/pxz7Pw"
-	//AlphaKey	SearchKey = "ALnHr9ZvSN7+hpON9YowQ27DOATgsJeZ/pxz7Pw"
+	//KevinKey SearchKey = "AO6VqIdnT4z+h5mO9Ig3QmvGMwTgsJeZ/pxz7Pw"
```

Comment:
Err, what it this? You've swaped from these two keys several times.

Reply:
this is an external token. Each token gives 10 access to the search service per day. I accidentally drained the first key when developing the frontend.

Reply:
We need this key to use the query API, but we can only use it 10 times a day **for free**, so we need to change it for test. AND DO NOT continue to ask questions about the key.